### ISSUE TITLE
feat: Ideas for DT Kit

### ISSUE THREAD

**Issue by arnoweiss** [eclipse-tractusx/eclipse-tractusx.github.io#334]
- [x] Elaborate on Provisioning and why it matters
- [x] setting asset/properties/@type when allowing access to multiple submodels
- [x] Comment on server-side filtering in EDC-Mgmt-Assets or AAS-Registry-API
- [x] Harmonize spelling, introduce abbreviations on each page and be consistent afterwards.
- [x] In DTR-EDC-Asset data structure: make sure Base-URL ends before /shell-descriptors or /lookup
- [ ] Remove explicit dependency to EDC --> won't do as DSP spec is too incomplete to give the full feature set in a generic manner.
- [ ] Tutorials for all three roles (opco, prov, cons) --> won't do as opco is not relevant, prov & cons will be handled by #728

**Comment by thomas-henn** [comment]
* In general, readability can be improved here and there, some typos and syntax glitches can be removed, tenses should be kept consistent.

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/c622ff831ff69004b7b9939e14f967c686ab0c25/docs-kits/kits/Digital%20Twin%20Kit/page_adoption-view.md?plain=1#L62
* What do you mean by expanding the API from different use cases? In which manner?

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/c622ff831ff69004b7b9939e14f967c686ab0c25/docs-kits/kits/Digital%20Twin%20Kit/page_adoption-view.md?plain=1#L65
* If you have no picture in mind, it his hard to follow the "noth-" and "southward" direction.

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/c622ff831ff69004b7b9939e14f967c686ab0c25/docs-kits/kits/Digital%20Twin%20Kit/page_adoption-view.md?plain=1#L77
* In general, avoid abbreviation(s), here: use "information" instead of "info"

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/c622ff831ff69004b7b9939e14f967c686ab0c25/docs-kits/kits/Digital%20Twin%20Kit/page_adoption-view.md?plain=1#L84
* What about citing the CX-0003 for Semantic Aspect Meta Models?

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/page_software-operation-view.md?plain=1#L21
* Not just BPN Discovery Servers. Also EDC discovery included.
* Should we rename "identifiers" to "specific asset identifiers", since actually this is what it is pointing to.
* I propose: "A microservice resolving a type of specific asset identifiers, *e.g.* `manufactuerPartId` against a set of BPN-, and EDC-Discovery Servers hosted at Catena-X operating companies."
* The same holds true in an analogue manner for the following descriptions

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/page_software-operation-view.md?plain=1#L28
* Is the meaning of abbreviations, *e.g.*, "OpCo", "IDTA", "IDSA", "AAS" known to the reader?
* Otherwise, please introduce those.

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L18
* Since the link is dead, please update the link.

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L25
* Since this is a citation, please cite the source as well.

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L27
* "What exactly is defined in the Catena-X standard CX - 0002."
* Is this a question or a reference to the previous sentence?
* If you refer to the previous sentence, I propose: "This subset is defined in ..."

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L33
* Name "SerialPartTypization" instead of "Nameplate".
* I suggest to refer to the description of the Semantic Aspect Meta Models (SAMM), *e.g.*, [SerialPartTypization](https://github.com/eclipse-tractusx/sldt-semantic-models/)

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L66
* Please adjust: "It allows to start a Discovery Process with only a type of AssetId, *e.g.*, `manufacturerPartId` and the actual content of an AssetId.
* We should think about if we cahnge AssetId and similar to `AssetId`?

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L67
* I do not understand the meaning of this sentence. Please clarify/simplify this sentence.

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L100
* You may rewrite this sentence: If this prior knowledge is given under all circumstances, Discovery Finder and BPN Discovery (steps 2-3) are not needed.

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L180
* What refers "She" to? I suggest: This data provider may create one ...

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L186
* Sentence incomplete. Maybe, it is not needed anymore? If needed please keep the tense, namely: "If a Data Provider wants to ..."

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L191
* That was quite right for the relase v3.1. From the v3.2 release onwards an data asset in the EDC can be a whole backend-system, which serves the content of submodels.
* This is the description for v3.1 https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Software%20Development%20View/Specification%20Traceability%20Kit/#data-provisioning: "EDC Asset ID: The EDC asset id under which the submodel was registered in the EDC. It must have the following format <AAS ID>-<Submodel ID>"

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L246
* Does "BPNL:" need to be part of the `ExternalReference:keys:[value]`?

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L254
* With or without leading `urn:uuid:`?

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L259
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L305
* `"type": "GlobalReference"` or `"type": "Submodel"`

______

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/b5f31baafac0af1ce0d1a8a29c73063936444a26/docs-kits/kits/Digital%20Twin%20Kit/Software%20Development%20View/page_software-development-view.md?plain=1#L344
* Is "DT" introduced as abbreviation to Digital Twin?
